aviary/docs/user_guide/reserve_missions.md

```diff
@@ -5,40 +5,65 @@
+The `"target_duration"` option creates a phase requiring the aircraft to fly for a specific amount of time.
+The `"target_distance"` option creates a phase requiring the aircraft to fly for a specific distance.
+Avoid using the optional flag if you have a reserve phase (i.e climb or descent) where you just want that phase to be completed as quickly as possible.
+The optional flags should not be combined as they will create overlapping constraints creating an infeasible problem.
 
-Reserve missions can be used to create fixed-time or a fixed-distance phases that are appended to the last phase of your nominal mission.
-To do this, set the `"reserve": True,` flag within the `user_options` for your reserve phases within `phase_info`.
-Additionally, if this is a fixed-time phase add `"target_duration"` and if it's a fixed-distance phase add `"target_distance"`.
-Reserve phases must be added to `phase_info` after the regular phases.
-You are not allowed to intersperse regular and reserve phases.
+You can chain together multiple reserve phases to make a complete reserve mission (i.e. climb to altitude, cruise for range, cruise for time, then descend).
+Examples of this are shown in `run_reserve_mission_multiphase.py` and `run_2dof_reserve_mission_multiphase.py`.
 
-The reserve phases will start at the same range and mass as the last regular phases, but the altitudes are not automatically connected.
+The reserve phases will start at the same range and mass as the last regular phases, but all other states (i.e. altitude, mach number) are not automatically connected.
 Thus you can fly climb, cruise, descent for regular phases and then immediately jump to an arbitrary altitude for the reserve mission.
 Or if you wanted to make things more realistic you could attach a climb phase and then add your reserve cruise.
-Make sure both the reserve climb and the reserve cruise phases both have the `reserve` flag within `phase_info` set to `True`.
+Make sure both the reserve climb and the reserve cruise phases both have `"reserve": True,` flag.
 
-You can chain together multiple reserve phases to make a complete reserve mission (i.e. climb to altitude, cruise for range, cruise for time, then descend).
-An example of this is shown in the example file `run_reserve_mission_multiphase_time_and_range.py`.
-You cannot create a reserve mission that enforces time or range constraints over multiple phases (i.e specify the total range covered by a climb + cruise + descent).
-
-## Examples
+### Examples
 
 Examples of single-phase and multi-phase reserve missions are presented in [Reserve Mission Examples](../examples/reserve_missions.md).
 
+### Caveats when using 2DOF
+
+If you are using 2DOF equations of motion (EOM) in your problem (i.e. `settings:equations_of_motion,2DOF`) there are some additional things you need to be aware of.
+The name of the reserve phase should include one of the keywords to indicate which EOM from 2DOF will be selected.
```

Comment:
Is the reserve phase required to start with "reserve_"?  Based on the check_phase_info code, it seems to be required, so we should be explicit about it.

Reply:
It doesn't have to start with "reserve_", it just has to be one of the keywords or one of the keywords with the prefix "reserve_" ("cruise" is a valid name for a reserve phase).
I think it would be reasonable to just say that reserve phases should be one of the keywords with the prefix.